src/amp.js

```diff
@@ -42,6 +43,7 @@ import {maybeTrackImpression} from './impression';
 try {
   // Should happen first.
   installErrorReporting(window);  // Also calls makeBodyVisible on errors.
+  installCacheServiceWorker(window);
```

Comment:
Does this call have to be in the initial critical path? Can it be deferred for a bit?